test/utils/testconfig/testconfig_test.go

```diff
@@ -157,36 +165,35 @@ func TestGetFullDiscoveredConfig(t *testing.T) {
 			args: args{namespace: "namespace1",
 				mode: OrdinaryClock},
 			want: TestConfig{
-				PtpModeDesired:            OrdinaryClock,
-				PtpModeDiscovered:         OrdinaryClock,
-				Status:                    DiscoverySuccessStatus,
-				DiscoveredSlavePtpConfig:  &ptpDiscoveryRes{Label: "label1", Config: *mockPtpConfig("config2", "namespace1", ptpv1.Slave, OrdinaryClock)},
-				DiscoveredMasterPtpConfig: &ptpDiscoveryRes{Label: "label1", Config: *mockPtpConfig("config1", "namespace1", ptpv1.Master, OrdinaryClock)},
+				PtpModeDesired:                    None,
+				PtpModeDiscovered:                 OrdinaryClock,
+				Status:                            DiscoverySuccessStatus,
+				DiscoveredClockUnderTestPtpConfig: (*ptpDiscoveryRes)(mockPtpConfig("config1", "namespace1", ptpv1.Slave, OrdinaryClock)),
+				DiscoveredClockUnderTestSecondaryPtpConfig: nil,
 			},
 		},
 		{
 			name: "Boundary clock",
 			args: args{namespace: "namespace1",
 				mode: BoundaryClock},
 			want: TestConfig{
-				PtpModeDesired:            OrdinaryClock,
-				PtpModeDiscovered:         BoundaryClock,
-				Status:                    DiscoverySuccessStatus,
-				DiscoveredSlavePtpConfig:  &ptpDiscoveryRes{Label: "label1", Config: *mockPtpConfig("config2", "namespace1", ptpv1.Slave, BoundaryClock)},
-				DiscoveredMasterPtpConfig: &ptpDiscoveryRes{Label: "label1", Config: *mockPtpConfig("config1", "namespace1", ptpv1.Master, OrdinaryClock)},
+				PtpModeDesired:                    None,
+				PtpModeDiscovered:                 BoundaryClock,
+				Status:                            DiscoverySuccessStatus,
+				DiscoveredClockUnderTestPtpConfig: (*ptpDiscoveryRes)(mockPtpConfig("config2", "namespace1", ptpv1.Slave, BoundaryClock)),
```

Comment:
can this be constant namespace1 and  config2

Reply:
fixed